tests/test_jwt.rs

```diff
@@ -1,10 +1,7 @@
 mod common;
 
 use crate::common::utils::{
-    load_scenario,
-    mock_jwk_issuer,
-    setup_mock_server,
-    TEST_JWKS_URL,
+    load_scenario, mock_jwk_issuer, setup_mock_server, TEST_JWKS_URL,
```

Comment:
I'm seeing that you ran `cargo fmt` already, so I'm not sure why the formatting is not correctly being applied, but the original formatting should be the correct one, unless I'm missing something.

Reply:
Huh weird